coolticket-frontend/src/components/Navigation.js

```diff
@@ -0,0 +1,46 @@
+import React, { Component } from "react";
+import { Link } from "react-router-dom";
+
+export default class Navigation extends Component {
+  render() {
+    return (
+      <nav className="navbar navbar-expand-lg navbar-dark bg-dark">
+        <div className="container">
+          <Link className="navbar-brand" to="/">
+            Coolticket
+          </Link>
+          <button
+            className="navbar-toggler"
+            type="button"
+            data-toggle="collapse"
+            data-target="#navbarNav"
+            aria-controls="navbarNav"
+            aria-expanded="false"
+            aria-label="Toggle navigation"
+          >
+            <span className="navbar-toggler-icon"></span>
+          </button>
+          <div className="collapse navbar-collapse" id="navbarNav">
+            <ul className="navbar-nav ml-auto">
+              <li className="nav-item active">
+                <Link className="nav-link" to="/api/v1/user">
+                  Create User <span className="sr-only">(current)</span>
+                </Link>
+              </li>
+              <li className="nav-item">
+                <Link className="nav-link" to="/api/v1/auth/login">
+                  Login <span className="sr-only">(current)</span>
+                </Link>
+              </li>
+              <li className="nav-item">
+                <Link className="nav-link" to="/api/v1/ticket">

```

Comment:
As I understand this URL is for requesting data, not to show a view (note it says "api")

Reply:
The same for the other ones